tests/integration/targets/test_mysql_user/tasks/test_user_locking.yml

```
@@ -0,0 +1,140 @@
+---
+
+- vars:
+    mysql_parameters: &mysql_params
+      login_user: '{{ mysql_user }}'
+      login_password: '{{ mysql_password }}'
+      login_host: '{{ mysql_host }}'
+      login_port: '{{ mysql_primary_port }}'
+
+  block:
+
+    # ========================= Prepare =======================================
+    - name: Mysql_user Lock user | Create a test database
+      community.mysql.mysql_db:
+        <<: *mysql_params
+        name: mysql_lock_user_test
+        state: present
+
+    # ========================== Tests ========================================
+
+    - name: Mysql_user Lock user | Create test user locked
+      community.mysql.mysql_user:
+        <<: *mysql_params
+        name: mysql_locked_user
+        host: '%'
+        password: 'msandbox'
+        locked: yes
```

```suggestion
        locked: yes
```
true/false is a recommended way since some time ago, officially. it's not critical here, but in modules docs and files it must be `true/false`